packages/analyzer/src/test/javascript/class-scanner_test.ts

```diff
@@ -64,33 +93,7 @@ suite('Class', async () => {
     if (class_.methods.size > 0) {
       result.methods = [];
```

Comment:
In this case the array methods are type safe, so we could golf this as:

```typescript
result.methods = [...class_.methods.values()].map(getMethod);
```

Reply:
Ah yes nice :smile: